src/com/hoffrogge/uebungen/KlassenWeitergeben.java

```diff
@@ -0,0 +1,25 @@
+package com.hoffrogge.uebungen;
+
+import java.util.ArrayList;
+
+import com.hoffrogge.lehreinheit02.Wuerfel;;
+
+public interface KlassenWeitergeben {
+	
+	public ArrayList<Wuerfel> instanceList = new ArrayList<Wuerfel>();
+	
+	/**
+	 * Diese Methode sollte einen W�rfel zu einer Liste von Rechtecksinstanzen hinzuf�gen.
```

Comment:
Stell' mal bitte die Datei auf Cp1252 um _oder_ entferne alle Umlaute. Ich bin zwar auch UTF-8 Fan, aber die Schulungslaptops haben in Eclipse alle Cp1252 drin, das bei jedem umzustellen ist gerdae nicht praktisch.

Reply:
Wieso Rechtecksinstanzen? copy paste Fehler?

Reply:
Jup. Sollten erst Rechtecke sein, vergessen zu ändern